en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+
+#### Glossary 
+
+- <a class="pricing" id="scheduler">Scheduler</a>: A scheduled task created on Choreo that runs every day.
+- <a class="pricing" id="oilPriceAPI">Oil Price API</a>: An external-facing API that returns the crude oil price for the day.
+- <a class="pricing" id="airFareDatabase">Air Fare Database</a>: The database that stores the jet fuel price which is used to compute the airfare for each route.
+- <a class="pricing" id="airlineRoutesAPI">Airline Routes API</a>: An external-facing API that  provides details about the airline routes and a list of airline routes.
+- <a class="pricing" id="airlineRoutesDatabase">Airline Routes Database</a>: The database that stores the airline routes information.
+
+<br/>
+
+#### Update jet fuel price daily
+
+![ Update jet fuel prices daily](../assets/img/references/scheduled-task-example1.png)
+
+Let’s see how we can calculate the steps for the update maintenance process:
+
+- The scheduled task ([scheduler](#scheduler)) on Choreo runs once per day.
+- The scheduler makes **1 API call to the [Oil Price API](#oilPriceAPI)** to  retrieve the daily oil price.  This is counted as **1 step**. 
+- Upon receiving the daily oil price, the Scheduler then applies the formula to calculate the jet fuel price and calculates daily jet fuel price.
+- The Scheduler on Choreo then makes **1 database call to the [Air Fare Database](#airFareDatabase)** to update the daily jet fuel price. This is counted as **1 step**. 
+
+
+- **Assuming the entire execution time for the update process takes 1200 ms**, the total number of steps can be calculated as follows:
+  
+        Number of API calls/database calls/events = 2 steps
+        Number of computation units = round((1200-500)/500) = 2 units = 2 steps
+        Total number of steps = 2 steps + 2 steps = 4 steps
+
+<br/>
+
+#### Update airfare daily  
+
+![ Update airfare daily](../assets/img/references/scheduled-task-example2.png)
+
+Let’s see how we can calculate the steps for the update process: 
+
+- The scheduled task ([scheduler](#scheduler))on Choreo runs once per day.
+- The Scheduler makes **1 API call to the [Airline Routes API](#airlineRoutesAPI)** to  retrieve the airline routes.  This is counted as **1 step**. 
+For ease of explanation, let’s *assume the above API call returns 5 airline routes*. 
+- Upon receiving the airline routes, the Scheduler then loops through the 5 airline routes and computes the airfare.  
+- The Scheduler on Choreo then makes **1 database call to the [Air Fare Database](#airFareDatabase)** to update the airfare for the airline routes. This is counted as **5 steps** as the database call was made for all airline routes received from the above API call.
```

Comment:
Change "as the database call" to "because the database call".

Reply:
done